obc.py

```diff
@@ -23,7 +23,12 @@ def start(self):
 
         # start apps
         imageService.start()
+<<<<<<< HEAD
```

Comment:
Looks like there actually was some merge conflicts in this file that we missed. You're gonna wanna go in here and remove `<<<<<<< HEAD`, `=======` and `>>>>>>> 4fc1f82be6b1373c222d146900e064766842638e`